src/GRA.Controllers/EventsController.cs

```diff
@@ -296,8 +332,15 @@ public IActionResult Index(EventsListViewModel model)
                 isCommunityExperience = true;
             }
 
+            int? page = null;
+            if (keepPage && model.PaginateModel?.CurrentPage > 1)
```

Comment:
This is weird. What are we doing here?

Reply:
Part of @iafb's changes

Reply:
It's for when updating a user's favorites without javascript. Typically that action is called for a new filter criteria which starts back at page 1 but when updating favorites we want to maintain the current page.